Submissions/@KaywebDev.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Caleb N",
+  "img": "https://avatars.githubusercontent.com/u/51725914?v=4",
+  "email": "caleb_d_n@sudomail.com",
+  "links": {
+    "website": "",
+    "linkedin": "https://www.linkedin.com/in/caleb-n-software/",
+    "github": "https://github.com/KaywebDev"
+  },
+  "jobTitle": "software engineer / web developer",
+  "location": {
+    "city": "",
+    "state": "MN",
+    "country": "united states"
```

Comment:
Please capitalize

Reply:
I have capitalized what I think needed it. Please let me know if there was something else that needed it.